### ISSUE TITLE
Android app do nothing

### ISSUE THREAD

**Issue by PrzemekSkw** [bauerj/paperless_app#112]
*Paperless App Version*: 0.1.2
*Do you use Paperless NG as a backend*: (Yes/No) No
Android 13
One Plus 9pro

Hello, today I install Paperless app from Google Play store. 
I try to add document from gallery but nothing happen.
I try to scan document and nothing was scanned.
I crop, check ✔️ and I still have empty app.
Is this app working normal or I have to add some permissions?
Regards.

**Comment by bauerj** [comment]
Hi,

this definitely doesn't sound normal. Are you able to add a document through the website and see it in the app?

**Comment by PrzemekSkw** [comment]
Yes, I see document when add in web browser. I noticed that now when I add with app I see document but I must close the app and open again.
